### ISSUE TITLE
GE 45600 remote fail to configure

### ISSUE THREAD

**Issue by vallon** [openhab/openhab#3215]
I tried to pair my GE 45600:

1) In habmin, "include" mode
2) On remote, setup-9-6-7.
3) Habmin shows a new node, but it is grey (uninitialized).
4) Cannot associate remote buttons to devices (setup-add-light-1 gives fast green flash, but pressing appliance pairing button does nothing - it continues to flash).

So, my thought is maybe the partial initialization is causing trouble, and would try to look for the cause.

I looked through the logs, but did not find anything glaring (I am a developer, but know nothing about openhab or zwave internals).

I tried attaching a file, but github complained.  I'll try again.
##

**Comment by vallon** [comment]
Don't know why I cannot attach a log file.  It said "Someting went really wrong, and we can't process that file".

Here is cut-and-paste:

```
2015-09-26 21:29:10.408 [DEBUG] [.b.z.i.p.s.AddNodeMessageClass:83  ]- NODE 13: Adding controller.
2015-09-26 21:29:37.712 [DEBUG] [b.z.i.protocol.ZWaveController:606 ]- NODE 13: Including node.
2015-09-26 21:29:37.713 [DEBUG] [b.z.i.protocol.ZWaveController:453 ]- NODE 13: Init node thread start
2015-09-26 21:29:37.716 [DEBUG] [.b.z.i.p.i.ZWaveNodeSerializer:138 ]- NODE 13: Serializing from file etc/zwave/node13.xml
2015-09-26 21:29:37.716 [DEBUG] [.b.z.i.p.i.ZWaveNodeSerializer:141 ]- NODE 13: Error serializing from file: file does not exist.
2015-09-26 21:29:37.717 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:267 ]- NODE 13: Node advancer - EMPTYNODE: queue length(0), free to send(true)
2015-09-26 21:29:37.717 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:333 ]- NODE 13: Node advancer: loop - EMPTYNODE try 1: stageAdvanced(false)
2015-09-26 21:29:37.717 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:338 ]- NODE 13: Node advancer: Initialisation starting
2015-09-26 21:29:37.717 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:866 ]- NODE 13: Node advancer - advancing to PROTOINFO
2015-09-26 21:29:37.717 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:333 ]- NODE 13: Node advancer: loop - PROTOINFO try 0: stageAdvanced(true)
2015-09-26 21:29:37.717 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:347 ]- NODE 13: Node advancer: PROTOINFO - send IdentifyNode
2015-09-26 21:29:37.717 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:240 ]- NODE 13: Node advancer - queued packet. Queue length is 1
2015-09-26 21:29:37.718 [DEBUG] [b.z.i.protocol.ZWaveController:545 ]- NODE 13: Init node thread finished
2015-09-26 21:29:37.723 [DEBUG] [i.p.s.IdentifyNodeMessageClass:52  ]- NODE 13: ProtocolInfo
2015-09-26 21:29:37.723 [DEBUG] [i.p.s.IdentifyNodeMessageClass:61  ]- NODE 13: Listening = false
2015-09-26 21:29:37.723 [DEBUG] [i.p.s.IdentifyNodeMessageClass:62  ]- NODE 13: Routing = false
2015-09-26 21:29:37.724 [DEBUG] [i.p.s.IdentifyNodeMessageClass:63  ]- NODE 13: Version = 3
2015-09-26 21:29:37.724 [DEBUG] [i.p.s.IdentifyNodeMessageClass:64  ]- NODE 13: FLIRS = false
2015-09-26 21:29:37.724 [DEBUG] [i.p.s.IdentifyNodeMessageClass:76  ]- NODE 13: Basic = Controller
2015-09-26 21:29:37.724 [DEBUG] [i.p.s.IdentifyNodeMessageClass:83  ]- NODE 13: Generic = Remote Controller
2015-09-26 21:29:37.724 [DEBUG] [i.p.s.IdentifyNodeMessageClass:90  ]- NODE 13: Specific = Portable Remote Controller
2015-09-26 21:29:37.724 [DEBUG] [.o.b.z.i.p.c.ZWaveCommandClass:224 ]- NODE 13: Creating new instance of command class NO_OPERATION
2015-09-26 21:29:37.724 [DEBUG] [.z.internal.protocol.ZWaveNode:523 ]- NODE 13: Adding command class NO_OPERATION to the list of supported command classes.
2015-09-26 21:29:37.724 [DEBUG] [.o.b.z.i.p.c.ZWaveCommandClass:224 ]- NODE 13: Creating new instance of command class BASIC
2015-09-26 21:29:37.724 [DEBUG] [.z.internal.protocol.ZWaveNode:523 ]- NODE 13: Adding command class BASIC to the list of supported command classes.
2015-09-26 21:29:37.725 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:1011]- NODE 13: Node advancer - PROTOINFO: Transaction complete (IdentifyNode:Request) success(true)
2015-09-26 21:29:37.725 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:201 ]- NODE 13: Node advancer - checking initialisation queue. Queue size 1.
2015-09-26 21:29:37.725 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:207 ]- NODE 13: Node advancer - message removed from queue. Queue size 0.
2015-09-26 21:29:37.725 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:267 ]- NODE 13: Node advancer - PROTOINFO: queue length(0), free to send(true)
2015-09-26 21:29:37.726 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:333 ]- NODE 13: Node advancer: loop - PROTOINFO try 1: stageAdvanced(false)
2015-09-26 21:29:37.727 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:866 ]- NODE 13: Node advancer - advancing to NEIGHBORS
2015-09-26 21:29:37.727 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:333 ]- NODE 13: Node advancer: loop - NEIGHBORS try 0: stageAdvanced(true)
2015-09-26 21:29:37.727 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:357 ]- NODE 13: Node advancer: NEIGHBORS - send RoutingInfo
2015-09-26 21:29:37.727 [DEBUG] [p.s.GetRoutingInfoMessageClass:29  ]- NODE 13: Request routing info
2015-09-26 21:29:37.727 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:240 ]- NODE 13: Node advancer - queued packet. Queue length is 1
2015-09-26 21:29:37.735 [DEBUG] [p.s.GetRoutingInfoMessageClass:47  ]- NODE 13: Got NodeRoutingInfo request.
2015-09-26 21:29:37.735 [DEBUG] [p.s.GetRoutingInfoMessageClass:71  ]- NODE 13: No neighbors reported
2015-09-26 21:29:37.736 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:1011]- NODE 13: Node advancer - NEIGHBORS: Transaction complete (GetRoutingInfo:Request) success(true)
2015-09-26 21:29:37.736 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:201 ]- NODE 13: Node advancer - checking initialisation queue. Queue size 1.
2015-09-26 21:29:37.736 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:207 ]- NODE 13: Node advancer - message removed from queue. Queue size 0.
2015-09-26 21:29:37.736 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:267 ]- NODE 13: Node advancer - NEIGHBORS: queue length(0), free to send(true)
2015-09-26 21:29:37.736 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:333 ]- NODE 13: Node advancer: loop - NEIGHBORS try 1: stageAdvanced(false)
2015-09-26 21:29:37.736 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:866 ]- NODE 13: Node advancer - advancing to FAILED_CHECK
2015-09-26 21:29:37.736 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:333 ]- NODE 13: Node advancer: loop - FAILED_CHECK try 0: stageAdvanced(true)
2015-09-26 21:29:37.737 [DEBUG] [i.p.s.IsFailedNodeMessageClass:31  ]- NODE 13: Requesting IsFailedNode status from controller.
2015-09-26 21:29:37.737 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:240 ]- NODE 13: Node advancer - queued packet. Queue length is 1
2015-09-26 21:29:37.741 [DEBUG] [i.p.s.IsFailedNodeMessageClass:54  ]- NODE 13: Is currently marked as healthy by the controller
2015-09-26 21:29:37.741 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:1011]- NODE 13: Node advancer - FAILED_CHECK: Transaction complete (IsFailedNodeID:Request) success(true)
2015-09-26 21:29:37.741 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:201 ]- NODE 13: Node advancer - checking initialisation queue. Queue size 1.
2015-09-26 21:29:37.742 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:207 ]- NODE 13: Node advancer - message removed from queue. Queue size 0.
2015-09-26 21:29:37.742 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:267 ]- NODE 13: Node advancer - FAILED_CHECK: queue length(0), free to send(true)
2015-09-26 21:29:37.742 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:333 ]- NODE 13: Node advancer: loop - FAILED_CHECK try 1: stageAdvanced(false)
2015-09-26 21:29:37.742 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:866 ]- NODE 13: Node advancer - advancing to WAIT
2015-09-26 21:29:37.742 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:333 ]- NODE 13: Node advancer: loop - WAIT try 0: stageAdvanced(true)
2015-09-26 21:29:37.742 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:380 ]- NODE 13: Node advancer: WAIT - Listening=false, FrequentlyListening=false
2015-09-26 21:29:37.742 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:398 ]- NODE 13: Node advancer: WAIT - Still waiting!
2015-09-26 21:29:55.145 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:997 ]- NODE 13: Node advancer - WAIT: The WAIT is over!
2015-09-26 21:29:55.145 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:201 ]- NODE 13: Node advancer - checking initialisation queue. Queue size 0.
```

Tried pairing, but didn't work.  Then, removed it:

```
2015-09-26 21:31:49.726 [DEBUG] [z.i.p.s.RemoveNodeMessageClass:76  ]- NODE 13: Removing controller.
2015-09-26 21:31:49.729 [DEBUG] [z.i.p.s.RemoveNodeMessageClass:80  ]- NODE 13: Removed from network.
2015-09-26 21:31:49.730 [DEBUG] [b.z.i.protocol.ZWaveController:617 ]- NODE 13: Excluding node.
```

**Comment by cdjackson** [comment]
I'm not familiar with the device, but it's a battery device and needs to be woken up a few times to complete initialisation. Currently there is no communication with the device...

**Comment by SkynetHome** [comment]
I have several GE switches in my home. I might be able to pick up one of these controllers to test and send back :)  Before I do that though, you might read through some of the comments on this amazon link where many people explain how they got it to work. If that doesn't help, let me know and I'll see what I can do.  http://www.amazon.com/GE-45600-Z-Wave-Handheld-compatible/product-reviews/B0013V6RW0/ref=cm_cr_dp_see_all_btm?ie=UTF8&showViewpoints=1&sortBy=bySubmissionDateDescending

**Comment by vallon** [comment]
I have tried various combinations of Setup-967, but no luck.  I have only tried trying to associate nodes with the GE controller (setup-add-switch), but the normal handshake does not work.

The controller was working with a Wink hub as a secondary controller.

A few months later, and there is no change in state.  One thing I noticed is that "All On/Off" works.  So, it seems the device is at least partially setup.

Habmin shows:

Status:
  Last Packet Sent: NEVER
  Last Packet Received: NEVER
  Heal Status: IDLE
  Node State: ALIVE PING {6 days ago}
  Packet Statistics: 0/0
  Dead: false
